### ISSUE TITLE
Goblin Gumbo Kettle

### ISSUE THREAD

**Issue by Cayo91** [dalaranwow/dalaran-wow#4271]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After you placed down the Kettle, got your goblin gumbo and consumed it, a buff is applied. Unfortunately that's it.

**Expected behaviour**: Tell us what should happen instead
Using this item creates a buff that makes you belch green breath at irregular intervals for five minutes (audio and visual effect).

**Steps to reproduce the problem**:

1. Put down a Goblin Gumbo Kettle
2. Consume it
3. Wait for 5 minutes

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Dont know how to proof it, because there isnt really something to screenshot.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

http://de.wowhead.com/item=33219

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/4193
